publish-backend/tests/helpers/strapi.js

```diff
@@ -0,0 +1,34 @@
+const Strapi = require("@strapi/strapi");
+const fs = require("fs");
+
+let instance;
+
+async function setupStrapi() {
+  if (!instance) {
+    await Strapi().load();
+    instance = strapi;
+
+    await instance.server.mount();
+  }
+  return instance;
+}
```

Comment:
What does `instance` do for us? It looks like it's being used to prevent duplicate inits, but it's never used again.

If that's the objective, we can do the following:
```suggestion
async function setupStrapi() {
  if (typeof strapi === "undefined") {
    await Strapi().load();
    await strapi.server.mount();
  }
}
```